spec/display-buffer-spec.coffee

```diff
@@ -703,6 +715,11 @@ describe "DisplayBuffer", ->
       expect(displayBuffer.screenPositionForBufferPosition([100000, 0])).toEqual [12, 2]
       expect(displayBuffer.screenPositionForBufferPosition([100000, 100000])).toEqual [12, 2]
 
+    it "clips correctly around atomic tab characters", ->
+      buffer.insert([0, 0], '\t')
```

Comment:
Good catch - this had been affected by my global tabLength change before, and when I took that commit out I missed this. I will add more tests here as well.